docs/guides/documentation/releases/gh_aggregate_release_notes.py

```diff
@@ -0,0 +1,99 @@
+import requests
```

Comment:
The script should probably include some comments at the top about dependencies. For example, it does `import requests` so needs a Python installation (or virtual environment) with [Requests](https://requests.readthedocs.io/) installed. Bonus points for mentioning a version number of `requests` too 👍  (same for `tqdm`, `yaml`).

Reply:
Echoes my comment above... Also, Python 3+.

Reply:
Also may want to add a little try-except error catching to explain to a casual user what's wrong. What if the template itself is munged up? That can be hard to catch. 

Some minimal logging (even just print statements) may be helpful, too.